addons/website_sale/static/src/js/website_sale_tour.js

```diff
@@ -0,0 +1,481 @@
+                model: 'res.config.settings',
+                method: 'create',
+                args: [{
+                    'auth_signup_uninvited': 'b2b',
+                    'show_line_subtotals_tax_selection': 'tax_included',
+                    'group_show_line_subtotals_tax_excluded': false,
+                    'group_show_line_subtotals_tax_included': true,
+                }],
+            });
+            var def2 = def1.then( function (resId) {
+                return rpc.query({
+                    model: 'res.config.settings',
+                    method: 'execute',
+                    args: [[resId]],
+                });
+            });
+            def2.then(function () {
+                window.location.href = '/web/session/logout';
+            });
+        },
+    },
+    // Testing b2b with Tax-Included Prices
+    {
+        content: "Click on shop",
+        extra_trigger: '.oe_login_form',
+        trigger: '#top_menu a[href="/shop"]:visible',
+    },
+    {
+        content: "Search product 'Storage box'",
+        extra_trigger: '.js_sale',
+        trigger: '#wrapwrap',
+        run: function () {
+            window.location.href = '/shop?search=Storage Box';
+        },
+    },
+    {
+        content: "Open product page",
+        trigger: '.oe_product_cart a:contains("Storage Box")',
+    },
+    {
+        content: "Add one more Storage Box",
+        trigger: '.js_add_cart_json:eq(1)',
+    },
+    {
+        content: "Check b2c Tax-Included Prices",
+        trigger: '.product_price .oe_price .oe_currency_value:containsExact(90.85)',
+    },
+    {
+        content: "Click on add to cart",
+        trigger: '#add_to_cart',
+    },
+    {
+        content: "Check for 2 products in cart and proceed to checkout",
+        extra_trigger: '#cart_products tr:contains("Storage Box") input.js_quantity:propValue(2)',
+        trigger: 'a[href*="/shop/checkout"]',
+    },
+    {
+        content: "Check Price b2c total",
+        trigger: 'tr#order_total_untaxed .oe_currency_value:containsExact(158.00)',
+    },
+    {
+        content: "Check Price b2c Sale Tax(15%)",
+        trigger: 'tr#order_total_taxes .oe_currency_value:containsExact(23.70)',
+    },
+    {
+        content: "Check Price b2c Total amount",
+        trigger: 'tr#order_total .oe_currency_value:containsExact(181.70)',
+    },
+    {
+        content: "Click on Login Button",
+        trigger: '.oe_cart a.btn:contains("Log In")',
+    },
+    {
+        content: "Submit login",
+        trigger: '.oe_login_form',
+        run: function (){
+            $('.oe_login_form input[name="login"]').val("abc@odoo.com");
+            $('.oe_login_form input[name="password"]').val("1admin@admin");
+            $('.oe_login_form').submit();
+        },
+    },
+    {
+        content: "Add new shipping address",
+        trigger: '.one_kanban form[action^="/shop/address"] .btn',
+    },
+    {
+        content: "Writing name in shipping address",
```

Comment:
Same remark, both writing step can be reduce in one run step in javascript, it will speed up the process and will still test the same things.